src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -50,6 +50,8 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
         private TaskCompletionSource<object> _cancelAllTcs;
         private CancellationTokenSource _cancelAllCts = new CancellationTokenSource();
 
+        public int ProcessId { get { return _process != null ? _process.Id : 0; } }
```

Comment:
Also, perhaps it's better to make it `int?`, and return `null` if there's no host.